.github/workflows/vc-command.yml

```diff
@@ -53,7 +53,7 @@ jobs:
               -vvvv --os-version $OS_VERSION \
```

Comment:
while being at it, how about quoting this as well?

Reply:
This one will never contain any spaces